index.html

```diff
@@ -16,12 +16,146 @@
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+              <ul class="nav justify-content-center">
+                <li class="nav-item">
+                  <a class="nav-link active" href="#">Background</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Hobbies</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Sports</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link disabled" href="#">Edication</a>
+                </li>
+              </ul>
+          </div>
+      </div>
+
+      <Div class="row picture-row">
+
+        <div class="col-sm-5">
+          <div class="card" style="width: 18rem;">
+            <img class="card-img-top" src="img/Hilal.jpg" alt="Card image cap">
+            <div class="card-body">
+              <p class="card-text">I have successfully designed, developed, and supported live use applications. I strive for continued excellence I provide
+              exceptional contributions to customer service for all customers</p>
+            </div>
+          </div>
+        </div>
+
+        
+
+
+        <div class=" col-sm-6 cover-letter" >
+          <h3>
+            Cover Letter:
+          </h3>
+          <p> As a successful IT GUY with experience leading teams to generate unsurpassed revenue growth, I am an extremely focused,
+          driven and results-oriented leader. I have extensive skills in the areas of revenue acceleration, digital market analysis, B2B sales,
+          and development of major online marketing initiatives. </p>
+           </div>
+
+          </Div>
+
+
+          <div class="row  row-badge">
+           <div class="col-12"> 
+             <button type="button" class="btn btn-primary">
+                 Notifications
+                <span class="badge badge-light">4</span>
+          </button>
+            </div>
           </div>
+
+
+
+
+
+           
+        
+
+      
+   <div class="row button-row">
+     <div class="col-12 ">
+
+       <a class="btn btn-primary" href="#" role="button">My-City</a>
+       <button class="btn btn-primary" type="submit">My-home</button>
+       <input class="btn btn-primary" type="button" value="My-School">
+       <input class="btn btn-primary" type="submit" value="My-Lab">
+       <input class="btn btn-primary" type="reset" value="My-Innovation">
+        
+      </div>
+   </div>
+
+   <div class="row jumbo-row">
+     <div class="col-12">
+
+      <div class="jumbotron" >
+        <img src="img/tech.jpg">
+        <h1 class="display-4">Hello Visitors, Hilal Aissan Here...!</h1>
+        <p class="lead">I CAN   SPEAK   JAVA  ,  JAVASCRIPT  , PAYTHON  , RUBY ,  CSS ,  HTML ,  C++  , C  , LESS ,  SASS ,  PHP  ...
+        <p>I am now seeking the opportunity to utilize my experience for your firm as a IT Manager.</p>
```

Comment:
Your `p` tags should contain one paragraph each and typically shouldn't be nested.

It's _Python_ by the way.